### ISSUE TITLE
[1.18.1] Conflict with Roughly Enough Items

### ISSUE THREAD

**Issue by Quizer9O8** [BelgianSalamander/MoreBundles#10]
After loading a new world and entering something in the search bar of REI the game crashes after a few seconds.
See crash log: https://crashy.net/KrlnIdbxm0yHiI0zQxtr

**Comment by BelgianSalamander** [comment]
I think this crash is caused by the same thing as the most recent crash given here (https://github.com/BelgianSalamander/MoreBundles/issues/7). I know what the mistake is and will fix it once I am able to.

**Comment by BelgianSalamander** [comment]
I have not been able to reproduce this issue but believe that it should be fixed in the latest version on curseforge (Fabric Beta 0.2.1 Bugfix 1). If you could confirm that this issue is fixed it would be greatly appreciated. Thanks :)

**Comment by Solbright** [comment]
I don't think this is fully fixed. Whenever I have MoreBundles installed, I can't use the search function in REI at all. Typing in the search box does nothing. Simply removing MoreBundles fixes this, but of course, no MoreBundles...

Using latest version Bugfix 2 for Fabric on 1.18.2.
